contracts/SpokePool.sol

```diff
@@ -1103,7 +1103,6 @@ abstract contract SpokePool is
         fillCounter[token] += _computeAmountPostFees(totalFillAmount, realizedLPFeePct);
     }
 
-    // The following internal methods emit events with many params to overcome solidity stack too deep issues.
```

Comment:
Have removed this comment because I left _emitFillRelay() as a convenience. My thinking is that since it dereferences a lot of members of relayExecution, it's more practical to keep it as-is.